build/images/base-windows/Dockerfile

```diff
@@ -61,3 +61,32 @@ COPY --from=windows-utility-base ["C:\\\\wins","C:\\\\wins"]
 COPY --from=windows-golang ["C:\\\\Program Files\\\\Go", "C:\\\\Program Files\\\\Go"]
 
 RUN setx /m PATH "C:\Program Files\7-Zip;C:\git\bin;C:\git\usr\bin;C:\mingw64\bin;C:\Program Files\Go\bin;%GOPATH\bin%;%PATH%"
+
+FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as windows-ovs
+ARG WIN_OVS_VERSION
+
+# Switch to a privileged user
+USER ContainerAdministrator
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+ENV WIN_OVS_VERSION ${WIN_OVS_VERSION}
+
+# Install VC
+RUN curl.exe -Lo C:\vc_redist.x64.exe https://aka.ms/vs/17/release/vc_redist.x64.exe; \
+    powershell -Command  "Start-Process -FilePath C:\vc_redist.x64.exe -ArgumentList ‘/install’,’/quiet’,’/norestart’ -Verb RunAs -Wait"
+
+# Download and extract OVS
+RUN curl.exe -fLo C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip https://downloads.antrea.io/ovs/ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip;  \
+    powershell -Command Expand-Archive C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip -DestinationPath C:\  ;  \
+    rm C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip
+
+# Download and extract OpenSSL
+RUN curl.exe -Lo C:\openssl-1.0.2u-x64_86-win64.zip https://indy.fulgan.com/SSL/openssl-1.0.2u-x64_86-win64.zip; \
+    powershell -Command Expand-Archive C:\openssl-1.0.2u-x64_86-win64.zip -DestinationPath C:\  ;  \
+    rm C:\openssl-1.0.2u-x64_86-win64.zip
+
+RUN cp C:/libeay32.dll C:/openvswitch/usr/sbin/ ; \
+    cp C:/ssleay32.dll C:/openvswitch/usr/sbin/ ; \
+    mv C:/libeay32.dll C:/openvswitch/usr/bin/ ; \
+    mv C:/ssleay32.dll C:/openvswitch/usr/bin/ ; \
```

Comment:
why does it place two replicas in sbin and bin? I see `Install-OVS.ps1` only uses sbin.

Reply:
ovs processes needs these dlls, without these libraries ovs processes does not start, since there are ovs exes in sbin and bin, we need to copy to both of these directories.

Reply:
How did it work before when it only installs these files in `sbin` previously?

Reply:
Earlier we verified and found that inside container, if these files are not available in both the folders,ovs processes fail to start.